common/constants/shared.ts

```diff
@@ -24,3 +24,10 @@ export const observabilityPluginOrder = 6000;
 export const UI_DATE_FORMAT = 'MM/DD/YYYY hh:mm A';
 export const PPL_DATE_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 export const PPL_INDEX_REGEX = /(search source|source|index)\s*=\s*([^|\s]+)/i;
+
+// Observability plugin URI 
+const BASE_OBSERVABILITY_URI = '/_plugins/_observability';
```

Comment:
are we refactoring notebooks backend plugin to provide this? or is it a new plugin

Reply:
I'm not sure about this. One thing would be, backward compatibility to keep in mind. @anirudha  had mentioned about keeping all documents in observability indexed together(differentiated by types) something similar to how ".kibana" index does.